tokens/src/components/Form/size.json

```diff
@@ -0,0 +1,172 @@
+{
+  "size": {
+    "form": {
+      "input": {
+        "height": {
+          "base": {
+            "value": "calc({typography.form.input.line-height.base.value} * 1em + {spacing.form.input.padding.y.base.value} * 2 + {size.form.input.height.border.value})",
+            "source": "$input-height"
+          },
+          "sm": {
+            "value": "calc({typography.form.input.line-height.sm.value} * 1em + {input.btn.padding.sm.y.value} * 2 + {size.form.input.height.border.value})",
+            "source": "$input-height-sm"
+          },
+          "lg": {
+            "value": "calc({typography.form.input.line-height.lg.value} * 1em + {input.btn.padding.lg.y.value} * 2 + {size.form.input.height.border.value})",
+            "source": "$input-height-lg"
+          },
+          "inner": {
+            "base": {
+              "value": "calc({typography.form.input.line-height.base.value} * 1em + {spacing.form.input.padding.y.base.value} * 2)",
+              "source": "$input-height-inner"
+            },
+            "half": {
+              "value": "calc({typography.form.input.line-height.base.value} * .5em + {spacing.form.input.padding.y.base.value})",
+              "source": "$input-height-inner-half"
+            },
+            "quarter": {
+              "value": "calc({typography.form.input.line-height.base.value} * .25em + calc({spacing.form.input.padding.y.base.value} / 2))",
+              "source": "$input-height-inner-quarter"
+            }
+          },
+          "border": { "value": "calc({size.form.input.width.border.value} * 2)", "source": "$input-height-border" }
+        },
+        "width": {
+          "hover": { "value": "0.063rem", "source": "$input-hover-width" },
+          "focus": { "value": "0.063rem", "source": "$input-focus-width" },
+          "border": { "value": "{input.btn.border.width.value}", "source": "$input-border-width" }
+        },
+        "border": {
+          "radius": {
```

Comment:
should the order of keys be inverted here? i.e. first `radius` and then `border`, like above with `height` and `width` attributes, they have `border` inside them instead of outside